src/sentry/plugins/sentry_webhooks/plugin.py

```diff
@@ -92,14 +90,7 @@ def get_group_data(self, group, event, triggering_rules):
         data['event'] = dict(event.data or {})
         data['event']['tags'] = event.tags
         data['event']['event_id'] = event.event_id
-        if features.has('organizations:legacy-event-id', group.project.organization):
```

Comment:
This feature should also be removed from the manager and the internal repository at some point along with or after this change

Reply:
Good point, I'll do it in 2 steps just to be safe

Reply:
There is one more reference of this feature in a test. `tests/sentry/plugins/sentry_webhooks/test_plugin.py:34`